BUILD.bazel

```diff
@@ -0,0 +1,309 @@
+        "include/maliput/api/rules/traffic_lights.h",
+        "include/maliput/api/segment.h",
+        "include/maliput/api/type_specific_identifier.h",
+        "include/maliput/api/unique_id.h",
+    ],
+    copts = ["-std=c++17"],
+    strip_include_prefix = "include",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":common",
+        ":math",
+    ],
+)
+
+cc_library(
+    name = "base",
+    srcs = [
+        "src/base/intersection.cc",
+        "src/base/intersection_book.cc",
+        "src/base/intersection_book_loader.cc",
+        "src/base/manual_discrete_value_rule_state_provider.cc",
+        "src/base/manual_phase_provider.cc",
+        "src/base/manual_phase_ring_book.cc",
+        "src/base/manual_range_value_rule_state_provider.cc",
+        "src/base/manual_right_of_way_rule_state_provider.cc",
+        "src/base/manual_rulebook.cc",
+        "src/base/phase_based_right_of_way_rule_state_provider.cc",
+        "src/base/phase_ring_book_loader.cc",
+        "src/base/phase_ring_book_loader_old_rules.cc",
+        "src/base/phased_discrete_rule_state_provider.cc",
+        "src/base/road_rulebook_loader.cc",
+        "src/base/road_rulebook_loader_using_rule_registry.cc",
+        "src/base/rule_filter.cc",
+        "src/base/rule_registry.cc",
+        "src/base/rule_registry_loader.cc",
+        "src/base/rule_tools.cc",
+        "src/base/traffic_light_book.cc",
+        "src/base/traffic_light_book_loader.cc",
+        "src/base/yaml_conversion.cc",
+    ],
+    hdrs = [
+        "include/maliput/base/intersection.h",
+        "include/maliput/base/intersection_book.h",
+        "include/maliput/base/intersection_book_loader.h",
+        "include/maliput/base/manual_discrete_value_rule_state_provider.h",
+        "include/maliput/base/manual_phase_provider.h",
+        "include/maliput/base/manual_phase_ring_book.h",
+        "include/maliput/base/manual_range_value_rule_state_provider.h",
+        "include/maliput/base/manual_right_of_way_rule_state_provider.h",
+        "include/maliput/base/manual_rulebook.h",
+        "include/maliput/base/phase_based_right_of_way_rule_state_provider.h",
+        "include/maliput/base/phase_ring_book_loader.h",
+        "include/maliput/base/phased_discrete_rule_state_provider.h",
+        "include/maliput/base/road_rulebook_loader.h",
+        "include/maliput/base/rule_filter.h",
+        "include/maliput/base/rule_registry.h",
+        "include/maliput/base/rule_registry_loader.h",
+        "include/maliput/base/rule_tools.h",
+        "include/maliput/base/simple_phase_ring_book.h",
+        "include/maliput/base/simple_rulebook.h",
+        "include/maliput/base/traffic_light_book.h",
+        "include/maliput/base/traffic_light_book_loader.h",
+        "include/maliput/base/yaml_conversion.h",
+    ],
+    copts = ["-std=c++17"],
+    strip_include_prefix = "include",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":api",
```

Comment:
Is the dependency to `common`  solved via `api` dependency?

Reply:
Good catch. It's explicitly dependend on, so the dependency should be explicit. Fixed.